modules/ROOT/pages/api-led-design.adoc

```diff
@@ -0,0 +1,159 @@
+
+Looking for an API specification that already does what you need is easy:
+
+* Look at the public Exchange. You’ll see some of the most popular API specifications, connectors, and other assets displayed.
++
+link:https://www.mulesoft.com/exchange["Take me there^", role="button-primary"]
+ +
+Select *REST APIs* from the *Type* dropdown to display only REST API specifications. Click on any specification to see the data types and the HTTP requests defined for the API. 
++
+* To look at just the items in your organization (account) Exchange, 
+log in to your Anypoint Platform account (your account).
++
+link:https://anypoint.mulesoft.com/login["Take me there^", role="button-primary"] 
+
+== Step 2: Create Your API Specification
+In link:https://anypoint.mulesoft.com/login/["Anypoint Platform^", role="button-primary"]:
+
+. Click the blue button under Design Center to open the API Designer.
+. Click *Create* to open the API Designer editor.
+. Click *Create API specification*.
+. Enter Hello-World for *Name* and leave the other default values.
+. Click *Create Specification*.
+. The API Designer editor displays starter text. Replace it with RAML that specifies the API:
+
+[source,raml]
+----
+#%RAML 1.0
+title: Hello, World
+version: v1
+description: A greeting for the world
+
+securitySchemes:
+ basic:
+   description: |
+     This API supports Basic Authentication.
+   type: Basic Authentication
+ 
+securedBy: [basic]
+
+types:
+ greeting: # <2>
+   properties:
+     todays-greeting: string
+
+/greeting:
+ /{todays-greeting}:
+     uriParameters:
+       todays-greeting: string
+     get: # <1>
+       responses: # <3>
+         200:
+           body:
+             application/json:
+               type: greeting
+         404: # <4>
+           body:
+             application/json:
+               properties:
+                 message: string
+               example: |
+                 {
+                   "message" : "Greeting not found"
+                 }
+----
+<1> A single HTTP request, GET
+<2> A single `dataType`, `greeting`, with a single property, `todays-greeting` (today’s greeting message). 
+<3> HTTP response
+<4> Error message and error handling
+
+== Step 3: Test Your API Specification
+Now that you’ve created a Hello World specification, test it by sending a request.
+The mocking service handles the creation of a functioning endpoint from your API specification
+and provides a simple UI for managing authentication, request headers, and response headers.
+
+To test your Hello World specification:
+
+. link:https://anypoint.mulesoft.com/designcenter/#/projects["Open the Hello World API specification that you created in Step 2.^" role="button-primary"]
+. Click the X next to *Mocking service* to enable the mocking service. If you see a checkmark instead of an X, the mocking service is already enabled.
```

Comment:
```suggestion
. Click the X next to *Mocking service* to enable the mocking service. 
If you see a checkmark instead of an X, the mocking service is already enabled.
```

Reply:
We should chat about these artificial line breaks. They should either be full paragraph breaks or not break at all.